README.md

```diff
@@ -187,6 +187,7 @@ Set up your GitHub Actions workflow with a specific version of your programming
 - [Export global environment variables for succeeding build steps](https://github.com/zweitag/github-actions)
 - [Programmatically set environment variables for use in subsequent steps](https://github.com/allenevans/set-env)
 - [Install Conda environments for Python](https://github.com/goanpeca/setup-miniconda)
+- [Setup NativeScript](https://github.com/marketplace/actions/setup-nativescript)
```

Comment:
The action lgtm and the category you put it in is great. The only issue is wrong link. Please switch the link to the repository: https://github.com/hrueger/setup-nativescript